Task/Task-WGSRareDiseaseTestOrderRequested-DirectToLab-Example.json

```diff
@@ -1,20 +1,24 @@
 {
   "resourceType": "Task",
   "id": "Task-WGSRareDiseaseTestOrderRequested-DirectToLab-Example",
-  "basedOn": [
-    {
-      "reference": "ServiceRequest/ServiceRequest-WGSTestOrderFormUpdated-DirectToLab-Example"
-    }
-  ],
   "status": "requested",
+    "statusReason": {
```

Comment:
Can remove status reason from this resource, the on hold example, detailing awaiting specimen/RoD is sufficient